chapters/data-management-plan.qmd

```diff
@@ -0,0 +1,119 @@
+
+## What is data
+
+Research data is any information that has been collected, observed, generated or created to validate original research findings. Examples of data could be interview recordings, experiment results, physical measurement,  notes from focus group's  meetings, notes from fieldwork, observations captured in photographs,  film or audio, text files extracted from a corpus, image of archival items or artworks, scraped websites, responses to survey questions.  Algorithms, code, scripts and software, simulations are often considered as research data. There is also physical data: samples, collections, artifacts. 
+
+Administrative documents, like informed consent forms and key files should be acknowledged as important elements of research data as well.  
+
+## Data Assets
+
+[https://libguides.vu.nl/rdm/dmp?#s-lg-box-wrapper-15125649 > Elisa has another look at the table description, because it is quite brief]
+
+Data assets are types of data, information collected or generated in the course of research activities. It is common to provide data assets based on data stage such as raw, processed, or analyzed. Raw Data refers to original data collected, Processed Data is data that has undergone some level of transformation or organization. Processing involves cleaning, formatting, and structuring raw data to make it more understandable and suitable for analysis. Analyzed Data usually results from statistical methods, detailed examination or interpretation.
+
+Here are some examples of data assets in research data management:
+
+| Data Stage     | Dataset description         | Type of data    | Format |
+|----------------|-----------------------------|-----------------|--------|
+| Raw data       | Interviews                  | Audio files     | MP3    |
+|                | Spectographic analysis      | Text files      | CSV    |
+| Processed data | Transcription of interviews | Word files      | Docx   |
+|                | Data spreadsheet            | SPSS files      | SAV    |
+| Analysed data  | Regression graphic          | Photoshop files | PSD    |
+|                | Data table                  | Word file       | Docx   |
+| Other          | Poster presentation         | Powerpoint      | PPS    |
+|                | Project Website             | HTML            |        |
+
+## DMP Elements
+
+[this is in the guidance on DMPonline, but we use this as a place where we can refer to ethics committees, privacy champions, etc. Make it clear that there is not necessarily a set order - sometimes there is, e.g. a DPIA may follow after a pre-DPIA indicates that]
+
+The VU DMP template consists of seven sections with questions. In DMPonline, there is guidance available for all sections, as well as example answers. When you are writing your DMP, you can consult this information directly in DMPonline.
+
+### Data description
+
```

Comment:
We still need to decide if there is any relevant information that should go here.